javascript/app-discover/src/domain/CausalEdgesReport.ts

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+import type { CausalDiscoveryAlgorithm } from './CausalDiscovery/CausalDiscoveryAlgorithm.js'
+
+export interface CausalEdgesReport {
+	source: string
+	target: string
+	correlation?: string
+	priorCausalKnowledge: Number
```

Comment:
type: 'constraint' | 'discovered'

Reply:
I also thought we could have a column is_constrained with a 0 or 1 value